lib/create-options.js

```diff
@@ -10,6 +10,12 @@ function createOptions(cwd, context) {
   options.env['npm_loglevel'] = 'error';
   options.env['npm_config_tmp'] = context.npmConfigTmp;
 
+  if (context.hasOwnProperty('module') && context.module.hasOwnProperty('envVar')) {
+    for (var i in context.module.envVar) {
```

Comment:
this will iterate across all properties in the prototype chain which may have unexpected consequences. We are already including lodash so it might makes it simpler to just use it.

```js
_.forEach(context.module.envVar, function (val, key) {
  options.env[key] = val;
});
```